tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/statistics/Statistics.java

```diff
@@ -295,4 +336,95 @@ public int serialize(OutputStream outputStream) throws IOException {
     return length;
   }
 
+  public static void main(String[] args) throws IOException {
```

Comment:
please change this main function as a UT function

Reply:
It is a temp test, should be removed.